### PR TITLE
Add basics of pre-WGS

### PR DESCRIPTION
**Pull request by apmasell**
This needs:

* better input data
* sliders that work
* to use Alexis's graph generation code

### CODE REVIEW

**Comment by callunity** [comment]
(I'm currently updating the RNA code to use Alexis's graph generation code, and am happy to help convert this one as well)

**Comment by apmasell** [comment]
I'll take a stab at it. I was mostly concerned about getting the right BamQC data.

File: application/templates/index.html [hunk 1]
```diff
@@ -6,7 +6,9 @@
 	<body>
 		<h1>Dashi</h1>
 		<h2>PRE-EXOME</h2>
-		<h3><a href="/preexome">PRE-EXOME Report</a></h3>
+		<h3><a href="/preqc-exome">PRE-EXOME Report</a></h3>
```
Comment (avarsava) [inline application/templates/index.html]: did the URL for the pre-exome report change?

File: application/dash_application/views/preqc_wgs.py [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    cache = QCETLCache()
+    if True:  # DELETE(amasella): once IchorCNA data is available
+        wgs_df = cache.bamqc.bamqc
+        wgs_df[ICHOR_COL.Ploidy] = numpy.NaN
+        wgs_df[ICHOR_COL.TumorFraction] = numpy.NaN
+        # This doesn't seem to exist even though it should
+        wgs_df[BAMQC_COL.MarkDuplicates_PERCENT_DUPLICATION] = numpy.NaN
+    else:
+        wgs_df = cache.bamqc.bamqc.merge(cache.ichorcna.ichorcna[[ICHOR_COL.Ploidy, ICHOR_COL.TumorFraction]], how="left", left_on=[
+            BAMQC_COL.Run, BAMQC_COL.Lane, BAMQC_COL.Barcodes], right_on=[ICHOR_COL.Run, ICHOR_COL.Lane, ICHOR_COL.Barcodes])
```
Comment (callunity) [inline application/dash_application/views/preqc_wgs.py]: Could this be spaced out a bit more (one item per line) for readability?

File: application/templates/index.html (hunk 1 above)
Comment (callunity) [reply application/templates/index.html]: ^ Good point. @apmasell if you're making this change, the `page_name` will have to be changed in `pre_exome.py`

Comment (apmasell) [reply application/templates/index.html]: I'm still very confused associations between URL, file name, `page_name` and that list.

Comment (callunity) [reply application/templates/index.html]: We should probably standardize them

**Pull request merged**